docs/en/operations/settings/index.md

```diff
@@ -7,90 +7,16 @@ pagination_next: en/operations/settings/settings
 
 # Settings Overview
 
-There are multiple ways to define ClickHouse settings. Settings are configured in layers, and each subsequent layer redefines the previous values of a setting.
+There are two main groups of ClickHouse settings:
 
-The order of priority for defining a setting is:
+- Global server settings
+- Query-level settings
 
-1. Settings in the `users.xml` server configuration file
+The main distinction between global server settings and query-level settings is that
```

Comment:
The main distinction between global server settings and session / query-level settings is that global server settings must be set in configuration files while session / query-level settings can be set in configuration files or with SQL queries.